roles/search_enterprise/defaults/main.yml

```
@@ -9,7 +9,7 @@ search_enterprise_repository: https://artifacts.alfresco.com/nexus/content/group
 
 search_enterprise_version: 4.2.0
 search_enterprise_zip_url: "{{ search_enterprise_repository }}/{{ search_enterprise_artifact_name }}/{{ search_enterprise_version }}/{{ search_enterprise_artifact_name }}-{{ search_enterprise_version }}.zip"
-search_enterprise_zip_checksum_url:  "sha1:{{ search_enterprise_zip_url }}.sha1"
+search_enterprise_zip_checksum:  "sha1:{{ search_enterprise_zip_url }}.sha1"
```

2 spaces after `:` hope it doesnt break anything
```suggestion
search_enterprise_zip_checksum: "sha1:{{ search_enterprise_zip_url }}.sha1"
```